keyboards/hasupad/lib/keylogger.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include "crkbd.h"
+
```

Comment:
This file does not appear to be used anywhere by the board

Reply:
Dear Sir, I want to achieve this